dspace-api/src/main/java/org/dspace/content/service/clarin/MatomoReportSubscriptionService.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content.service.clarin;
```

Comment:
@kuchtiak-ufal why is the interface and its implementation in a different package?

Reply:
ok, I haven't noticed that